### DISCUSSION TITLE
TRSケーブルで繋いだ子キーボードの入力が受け付けられない

### DISCUSSION THREAD

**Discussion by mushuper** [Yowkees/keyball#610] [Q&A]
いつもお世話になっております。
１年程度愛用させていただいております。

この度、
https://github.com/Yowkees/keyball/discussions/301
↑と似た状況になり、自己解決を試みましたがうまくいかず、質問させていただければと思います。

### 【状況】
- 子キーボードのキー入力、トラックボールの動作が反映されない
- TRSケーブル接続時に、子キーボード側の液晶には「keyball」ロゴは表示される
- 親キーボードの液晶に、親キーボードへの入力内容は表示されるが、子キーボードのキー入力、トラックボール入力は表示されない
- 親を入れ替えても同じ現象
- 上記の状況が数ヶ月に一度発生し、その度にUSBやTRSケーブルの抜き差しで修復していたが、今回は２日連続発生したので取り急ぎ右手のみ修復を試みたところ、状況が悪化し完全に親子が分断された

### 【環境】
- keyball61
- 右手トラックボール
- 親キーボードは左手

### 【実施したこと】
上記URLを参考に、DATA線を再度はんだ付け（ただし、ハンダをうまく吸い取れず中途半端な作業になってしまっている気がします…。）

状況から、右側に原因があるだろうと推測しています。
![IMG_1043](https://github.com/Yowkees/keyball/assets/140997560/e126af92-05ad-4996-a362-0a8146804297)

ご教示いただけますと幸いです。

**Comment by Ryo624** [comment]
見間違いでしたら申し訳ないのですがDATA線のはんだが少し隣の端子に触れているように見えるので触れないようにできますでしょうか。
また、TRRSジャックのはんだが上手く端子に馴染めていない箇所が見えますので、新しいはんだを少し加えながらやり直してみてください。
